.github/workflows/build.yml

```diff
@@ -9,11 +9,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        rails: ["7.0"]
-        ruby: [3.2]
+        rails: ["7.0", "6.1"]
+        ruby: ["3.2", "3.1"]
         include:
-          - rails: "6.1"
-            ruby: "3.1"
```

Comment:
this essentially changes nothing... rails 6.1, ruby 3.1 was already there

Reply:
Oh, I guess it adds 7.0/3.1 and 6.1/3.2

Reply:
Yeah, those are the ones I was going for. We'd probably have been fine without them, but it doesn't hurt.

Reply:
clearly, I like this pattern. I started using include instead of exclude recently. makes things much nicer. and with the quick matrix for the top two versions, its even better.

Reply:
I like it too!